abaco_test.go

```diff
@@ -78,7 +78,7 @@ func TestGeneratePackets(t *testing.T) {
 	}
 }
 
-func _skip_TestAbacoRing(t *testing.T) {
+func TestAbacoRing(t *testing.T) {
```

Comment:
I don't understand why these were skipped at all. Did you commit to Master without me noticing? In my version of master they aren't skipped.

Reply:
I must have. They crash a lot, so when I was running tests in the debugger, I had to skip them to actually catch the crashes I was interested in in the EMT code.

Reply:
Bad Galen! Good for restoring them.

Reply:
No, bad tests!!! Brilliant skipping to make progress!